pstore/config/api.go

```diff
@@ -83,7 +78,8 @@ func Reset(configs ...Config) {
 
 // Decorator creates a decorated writer.
 type Decorator struct {
-	// Maximum reocrds to write per minute. 0 or negative means no limit.
+	// Maximum reocrds to write per minute. Optional.
+	// 0 or negative means no limit.
```

Comment:
Fair enough. I will add this as  JIRA ticket. I will also create JIRA tickets for Eric's two suggestions from my demo.